modules/swagger-codegen/src/main/resources/JavaPlayFramework/newApi.mustache

```diff
@@ -3,11 +3,12 @@ package {{package}};
 {{#imports}}import {{import}};
 {{/imports}}
 
-import play.mvc.Http;
 import java.util.List;
 import java.util.ArrayList;
-import java.util.HashMap;
```

Comment:
Here too we can't remove, if we have a collection which is a hash, it will not compile